python/ray/serve/tests/test_config_files/three_deployments.yaml

```diff
@@ -0,0 +1,61 @@
+deployments:
+
+  - name: shallow
+    version: None
+    prev_version: None
```

Comment:
as we discussed in the review, let's drop version and prev_version from the REST API